sys/targets/targets.go

```diff
@@ -152,41 +163,47 @@ var List = map[string]map[string]*Target{
 			PtrSize:          4,
 			PageSize:         4 << 10,
 			Int64Alignment:   4,
-			CFlags:           []string{"-m32"},
-			CCompilerPrefix:  "x86_64-linux-gnu-",
+			cflagsCommon:     []string{"-m32"},
+			Triple:           "x86_64-linux-gnu",
 			KernelArch:       "i386",
 			KernelHeaderArch: "x86",
 		},
 		"arm64": {
 			PtrSize:          8,
 			PageSize:         4 << 10,
-			CCompilerPrefix:  "aarch64-linux-gnu-",
+			Triple:           "aarch64-linux-gnu",
 			KernelArch:       "arm64",
 			KernelHeaderArch: "arm64",
 		},
 		"arm": {
-			VMArch:           "arm64",
-			PtrSize:          4,
-			PageSize:         4 << 10,
-			CFlags:           []string{"-D__LINUX_ARM_ARCH__=6", "-march=armv6"},
-			CCompilerPrefix:  "arm-linux-gnueabi-",
+			VMArch:       "arm64",
+			PtrSize:      4,
+			PageSize:     4 << 10,
+			cflagsCommon: []string{"-D__LINUX_ARM_ARCH__=6"},
+			cflagsCompiler: map[string][]string{
+				"gcc": []string{"-march=armv6"},
```

Comment:
These both seem to be clang flags as well:
```
$ clang /tmp/test.c --target=arm-linux-gnueabi -march=armv6 && echo OK
OK
$ clang /tmp/test.c --target=mips64el-linux-gnuabi64 -march=mips64r2 && echo OK
OK
```
Perhaps we can get away without per-compiler flags here at all?

Reply:
You are right. Maybe I've been attempting to compile without --target, that's why -march didn't work. I think we can indeed go without the per-compiler flags.